sql-plugin/src/main/scala/org/apache/spark/sql/rapids/datetimeExpressions.scala

```diff
@@ -877,6 +878,74 @@ case class GpuFromUnixTime(
   override lazy val resolved: Boolean = childrenResolved && checkInputDataTypes().isSuccess
 }
 
+
+class FromUTCTimestampExprMeta(
+    expr: FromUTCTimestamp,
+    override val conf: RapidsConf,
+    override val parent: Option[RapidsMeta[_, _, _]],
+    rule: DataFromReplacementRule)
+  extends BinaryExprMeta[FromUTCTimestamp](expr, conf, parent, rule) {
+
+  private var timezoneIsNull : Boolean = false
+
+  override def tagExprForGpu(): Unit = {
+    val timezoneShortID = extractStringLit(expr.right).getOrElse("")
```

Comment:
Why does this default to `""` which will result in a confusing exception later from `ZoneId.of`?  Shouldn't this throw IllegalStateException or something, as it should always be a literal due to the tagging checks?